client/my-sites/customer-home/cards/features/help-search/search-card.jsx

```diff
@@ -103,9 +87,23 @@ const mapStateToProps = ( state, ownProps ) => ( {
 	selectedResultIndex: getSelectedResultIndex( state ),
 	selectedResult: getInlineHelpCurrentlySelectedResult( state ),
 } );
+
+const requestInlineSearchResultsAndTrack = ( searchQuery ) =>
+	( ! searchQuery || ! ( searchQuery.trim() ).length )
+		? requestInlineHelpSearchResults()
+		: withAnalytics(
+			composeAnalytics (
+				recordTracksEvent( 'calypso_inlinehelp_search', {
```

Comment:
@retrofox I don't have super strong feelings on this, but I do think it'd probably make sense for this one to be `calypso_customer_home_support_card_search` or similar and drop the location prop.

Were folks not interested in a bump stat?

Reply:
Thanks, Kerry for your answer.

> I do think it'd probably make sense for this one to be calypso_customer_home_support_card_search or similar for discoverability and drop the location prop.

The goal for our purpose is what's the best way to compare stats between these two components. Using an event property allows us to segment the stats which, I think, makes things easier. Compares different events maybe could get trickier.

<img src="https://user-images.githubusercontent.com/77539/83189199-3dafdd00-a107-11ea-8ce3-8632c76af2dc.png" width="400px" />

> We could keep them grouped if we were interested in comparing searches from inline vs customer home.

Could you elaborate on this idea, please?

> Also, were folks not interested in a bump stat?

AFAIK, we haven't added a bump stats in the current Inline Help Popover component, so for this reason I hadn't added to this one either. Asking just in case...

Reply:
> Could you elaborate on this idea, please?

😆 you just screencapped that yes, you would like to compare searches. If that's the case feel free to keep the grouping but you may want to update documentation for folks.